purchase_order_maintenance_equipment_link/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="maintenance_equipment_form_view" model="ir.ui.view">
+        <field name="name">maintenance.equipment.form.view (in purchase_order_maintenance_equipment_link)</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button 
+                    name="action_view_purchase_order" 
+                    type="object" 
+                    class="oe_stat_button" 
+                    icon="fa-shopping-cart" 
+                    groups="purchase.group_purchase_user"
+                    >
+                    <field string="Purchase Orders" name="count_purchase_order_ids" widget="statinfo" />
```

Comment:
A bit long, it could be reduced:
```suggestion
                    <field string="Purchases" name="count_purchase_order_ids" widget="statinfo" />
```